epsagon/runners/python_function.py

```diff
@@ -37,21 +39,36 @@ def __init__(
         super(PythonRunner, self).__init__(start_time)
 
         self.event_id = str(uuid.uuid4())
-        self.resource['name'] = wrapped_function.__name__
+        self.resource['name'] = name if name else wrapped_function.__name__
         self.resource['operation'] = self.OPERATION
+
         self.resource['metadata'].update({
-            'module': wrapped_function.__module__,
-            'args_length': len(wrapped_args),
-            'kwargs_length': len(wrapped_kwargs)
+            'python.module': wrapped_function.__module__,
```

Comment:
Does this renaming of the metadata require any adaptation in the Epsagon backend as well?

Reply:
Nope